app/controllers/inventories_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
 
   def show
     @user = current_user
+    # @inventory = Inventory.find(params[:id])
```

Comment:
- great job addressing the change, kindly remove this commented piece of code to keep your codebase clean and maintainable

Reply:
## TESTS 
- As suggested by the previous reviewer, your tests are failing. please, fix them. If your test is passing from your end, kindly add a screenshot in this comment.

Reply:
**Integration tests:**
![image](https://github.com/katarighe/recipe-app/assets/80690364/0be60cc5-2f3e-4a6b-86df-fe7cb8709b48)

**Unit tests:**
![image](https://github.com/katarighe/recipe-app/assets/80690364/e2629a88-4416-4a04-b74c-56a0994269d6